data/blog/webview2.md

```diff
@@ -0,0 +1,108 @@
+---
+title: WebView2 and Electron
+author:
+- electron
+date: '2021-07-08'
+---
+
+Over the past week, we’ve received several questions about the differences between the new [WebView2](https://docs.microsoft.com/en-us/microsoft-edge/webview2/) and Electron.
+We want to provide some early information on the similarities and differences of the two solutions.
+
+---
+
+## Architecture Overview
+
+Electron and WebView2 both build from the Chromium source for rendering web content.
+Strictly speaking, WebView2 builds from the Edge source, but Edge is built using a fork of the Chromium source.
+Neither Electron nor WebView2 share any DLL files on the system with Edge or Chrome.
```

Comment:
Evergreen WebView2 Runtime binaries do actually hard link with Edge (Stable channel as of Edge 90). This means they share disk and some working set. We haven't publicly documented this currently in our WebView2 developer docs, so I don't have a good public doc link for you to point to as reference (though maybe I should).

Reply:
Very cool. I'll update the info, and if you have a link before publish we can include it.